cgroups/cgroups.go

```diff
@@ -0,0 +1,104 @@
+package cgroups
+
+import (
+	"bufio"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+
+	rspec "github.com/opencontainers/runtime-spec/specs-go"
+)
+
+var (
+	// AbsCgroupPath is absolute path for container's cgroup mount
+	AbsCgroupPath = "/cgrouptest"
```

Comment:
Do we need to put these two variables into the cgroups pkg?
Beside these, the PR looks good to me.

Reply:
We can change it later if there is a better place to leave these.